CODE_OF_CONDUCT.md

```diff
@@ -62,6 +64,16 @@ affect a person's ability to participate within them.
   constructively. Remember that we are different. Different people
   have different perspectives on issues. Being unable to understand why someone
   holds a viewpoint doesn't mean that they're wrong.
+- **Avoid excessively long messages.**
+  Long messages take more time for others to read and respond to, and
+  they risk addressing multiple issues at once, making it harder to
+  address everything comprehensively. Short, concise messages are more
+  efficient and help prevent communication misunderstandings.
+- **Avoid pressing for answers.**
+  The Coq community is generally highly responsive to users and
+  developers questions. However, in case you don't receive
+  an answer to your interogation, there is no need to push for
+  response. 
```

Comment:
I imagine people reading this will not see their own questions as an interrogation, and this sentence reads to me as assuming by default that the reader is wrong.  I'd rather see a more constructive version ("If you feel your questions or concerns are not receiving timely / adequate responses, instead of taking it up with the individual developers, who may feel XYZ, instead please use ABC process to raise your concerns.  If you experience someone else's questions or messages as excessively JKL, you can use process DEF.")

Reply:
See my response in https://github.com/coq/coq/pull/19812#discussion_r1832381658.